### PR TITLE
Add Admonitions and a Document section

### PR DESCRIPTION
**Pull request by haotian2006**
Added Admonis 
image
image

Docs section (going to change later)

image

### CODE REVIEW

**Comment by haotian2006** [comment]
@cyrus01337 for some reason the formatter thinks there's a syntax error in the mdx and pastor file and I'm unsure how to fix it

**Comment by cyrus01337** [comment]
Hmm, very strange, and your current files don't show what it's complaining about either. I'll take a look.

File: src/components/Aside.astro [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+<div>
+  {!collapsed && (
+    <details open>
+      <summary class="flex items-center gap-2 pb-2">
+        <Icon name={icon} class="w-6 h-6" />
+        <p class="text-sm font-bold">{text ? text : variant.toUpperCase()}</p>
+        <span class="icon"></span>
+      </summary>
+      <slot />
+    </div>
```
Comment (cyrus01337) [inline src/components/Aside.astro:54]: Remove - there's no 2nd div, causing invalid HTML.

File: src/components/Aside.astro [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+  )}
+  {collapsed && (
+    <details>
+      <summary class="flex items-center gap-2 pb-2">
+        <Icon name={icon} class="w-6 h-6" />
+        <p class="text-sm font-bold">{text ? text : variant.toUpperCase()}</p>
+        <span class="icon"></span>
+      </summary>
+      <slot />
+    </div>
```
Comment (cyrus01337) [inline src/components/Aside.astro:65]: Remove - there's no 2nd div, causing invalid HTML.

**Review by cyrus01337 — CHANGES_REQUESTED** [review]
The MDX files can be formatted via `npm run format`, you can verify that the project is formatted via `npm run lint:formatting`. The pipeline will always tell you if you haven't formatted your current changes, hence the error from before.

**Review by cyrus01337 — APPROVED** [review]
These look so cool.

**Pull request merged**